L1Trigger/L1TMuonOverlapPhase1/src/Omtf/GoldenPatternResult.cc

```diff
@@ -316,6 +321,123 @@ void GoldenPatternResult::finalise9() {
   //by default result becomes valid here, but can be overwritten later
 }
 
+void GoldenPatternResult::finalise10() {
+  for (unsigned int iLogicLayer = 0; iLogicLayer < stubResults.size(); ++iLogicLayer) {
+    unsigned int connectedLayer = omtfConfig->getLogicToLogic().at(iLogicLayer);
+
+    if (omtfConfig->isBendingLayer(iLogicLayer)) {  //the DT phiB layer is counted only when the phi layer is fired
+      if (firedLayerBits & (1 << iLogicLayer)) {
+        if (firedLayerBits & (1 << connectedLayer)) {
+          firedLayerCnt++;
+          pdfSum += stubResults[iLogicLayer].getPdfVal();
+        } else {
+          firedLayerBits &= ~(1 << iLogicLayer);
+          stubResults[iLogicLayer].setValid(false);
+          //if(stubResults[iLogicLayer].getPdfVal() == 0) pdfSum -= 64;; //there was hit, but it did not fire to the pdf - this is not possible here, since the banding layer if fired here
+          //so in this case simply:
+          //pdfSum += 0;
+        }
+      } else {
+        //bending layer fired, but not fits to the pdf, N.B works only with the patterns having "no hit value" and with noHitValueInPdf = True
+        /*if (stubResults[iLogicLayer].getPdfVal() == 0)
+          pdfSum -= 32; //has no sense with extrapolation from the ref layer using the phiB
+        else*/
+        pdfSum += stubResults[iLogicLayer].getPdfVal();  //bending layer not fired at all
+      }
+    } else {
+      if (iLogicLayer < 10 && stubResults[iLogicLayer].getPdfVal() == 0)
+        pdfSum -= 32;
```

Comment:
Would you please remove some of the commented code unless it serves a demonstrative purpose, and maybe leave a comment explaining where `10` and `32` are coming from here?